pkgs/by-name/li/libxmlxx5/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  libxml2,
+  glibmm,
+  meson,
+  ninja,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libxmlxx5";
+  version = "5.4";
+
+  src = fetchurl {
+    url = "https://download.gnome.org/sources/libxml++/${version}/libxml++-${lib.versions.pad 3 version}.tar.xz";
+    sha256 = "sha256-6aI8Q2aGqUaY0hOOa8uvhJEh1jv6D1DcNP77/XlWaEg=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    ninja
+  ];
+
+  buildInputs = [ glibmm ];
+
+  propagatedBuildInputs = [ libxml2 ];
+
+  doCheck = true;
```

```suggestion
  doCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
```
https://github.com/NixOS/nixpkgs/blob/c3b4161dac4a30a098a5c35767b1f0b41916d52f/pkgs/stdenv/generic/make-derivation.nix#L286-L288